### PR TITLE
[IMP] account_statement_base_import: Fill balance start when parser doesn't filled it

### PR DESCRIPTION
**Pull request by pedrobaeza**
Get starting balance from journal balance if parser doesn't fill this data, simulating the manual flow

### CODE REVIEW

**Comment by guewen** [comment]
:+1:

**Comment by gurneyalex** [comment]
:+1:

File: __unported__/account_statement_base_import/statement.py [hunk 1]
```diff
@@ -127,13 +127,22 @@ def prepare_statement_lines_vals(
...
-    def prepare_statement_vals(self, cr, uid, profile_id, result_row_list, parser, context):
+    def prepare_statement_vals(self, cr, uid, profile_id, result_row_list,
+                               parser, context=None):
         """
         Hook to build the values of the statement from the parser and
         the profile.
         """
         vals = {'profile_id': profile_id}
         vals.update(parser.get_st_vals())
+        if not vals.get('balance_start'):
```
Comment (sbidoul) [inline __unported__/account_statement_base_import/statement.py:138]: Is it better to use if vals.get('balance_start') is None? To distinguish cases were the parser returns a 0 balance from cases were the parser does not return balance_start.

**Comment by sbidoul** [comment]
Oh, I notice this one is already merged in 7.0 and propagated to 8.0. So this PR should be closed?

**Comment by pedrobaeza** [comment]
Yeah, I also didn't notice that. BTW, I prefer this method because this balance is filled by default with 0.0, so the method `if ... is None` is not valid.

**Pull request closed without merging**